app/javascript/dashboard/components/ChatList.vue

```diff
@@ -168,8 +176,12 @@
         v-if="showAdvancedFilters"
         :initial-filter-types="advancedFilterTypes"
         :initial-applied-filters="appliedFilter"
+        :active-folder-name="activeFolderName"
         :on-close="closeAdvanceFiltersModal"
+        :is-folder-view="hasActiveFolders"
         @applyFilter="onApplyFilter"
+        @updateActiveFolderName="onUpdateFolderName"
+        @updateFolder="onUpdateSavedFilter"
```

Comment:
```suggestion
        @updateActiveFolderName="onUpdateFolderName"
        @updateFolder="onUpdateSavedFilter"
```

This can be a single update action that can be triggered instead of two separate ones, this way you won't have to save the `updatedFolderName` variable. In principle, the state should not be copied in the parent, it should just stay with the owner component.

The `updateSavedCustomViews` in the component can pass both the filters as well as the the name of the folder